src/integration/HiddenWindow.cpp

```diff
@@ -0,0 +1,31 @@
+#include "pch/stdafx.h"
+#include "integration/HiddenWindow.h"
+#include "integration/ExternalMessageEventHandler.h"
+
+namespace UKControllerPlugin {
+    namespace Integration {
+        ExternalMessageEventHandler* handler = nullptr;
+
+        LRESULT CALLBACK HiddenWindow(HWND hwnd, UINT msg, WPARAM wParam, LPARAM lParam)
+        {
+            switch (msg) {
+                case WM_CREATE: {
+                    handler = reinterpret_cast<ExternalMessageEventHandler*>(
+                        reinterpret_cast<CREATESTRUCT*>(lParam)->lpCreateParams
+                    );
+                    return TRUE;
+                }
+                case WM_COPYDATA: {
+                    COPYDATASTRUCT* data = reinterpret_cast<COPYDATASTRUCT*>(lParam);
+
+                    if (data != nullptr && data->dwData == 811 && data->lpData != nullptr && handler != nullptr) {

```

Comment:
811 seems a little magic to me 🗡️

Reply:
You just have to pick a number so I came up with a random one :D This is the one we've sent to the vstrips team, so probably best we don't change it if we can help it :)

Reply:
Can I have a comment or constant to that effect?

Reply:
Done!